### PR TITLE
[UK-943]Add open and group samples

### PR DESCRIPTION
**Pull request by sravan-s**
Add couple of samples to show how real world project might look like

### CODE REVIEW

File: samples/openchannel/src/components/StreamingChannelList.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+function StreamingChannelList({
+  sdk,
+  user,
+  currentChannelUrl,
+  setCurrentChannel
+}) {
+  const [channels, setChannels] = useState([]);
+  useEffect(() => {
+    if (!sdk || !sdk.OpenChannel) {
```
Comment (HoonBaek) [inline samples/openchannel/src/components/StreamingChannelList.jsx:16]: same here, could you check if the createOpenChannelListQuery exists?

**Review by HoonBaek — COMMENTED** [review]
`samples/groupchannel/src/styles.js` seems that It doesn't need to be.

File: samples/groupchannel/src/const.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// put your own APP_ID here
+// get your app_id -> https://dashboard.sendbird.com/auth/signin
+export const APP_ID = "2D7B4CDB-932F-4082-9B09-A1153792DC8D";
+
+// set your own USER_ID and NICKNAME
+export const USER_ID = "sendbirdian-200720";
+export const NICKNAME = "Sendbirdian84";
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply samples/groupchannel/src/const.js:7]: done

[the remark this answers is not in the document]
Comment (sravan-s) [reply samples/groupchannel/src/const.js:7]: I was looking to setup using  https://create-react-app.dev/docs/adding-custom-environment-variables/ `.env` but somehow its not working. So, I made a temporary workout

**Review by HoonBaek — APPROVED** [review]
LGTM

**Pull request merged**